library/core/src/cmp.rs

```diff
@@ -916,14 +916,18 @@ pub macro Ord($item:item) {
 /// easy to accidentally make them disagree by deriving some of the traits and manually
 /// implementing others.
 ///
-/// The comparison must satisfy, for all `a`, `b` and `c`:
+/// The comparison relations must satisfy the following conditions
+/// (for all `a`, `b`, `c` of type `A`, `B`, `C`):
 ///
-/// - transitivity: `a < b` and `b < c` implies `a < c`. The same must hold for both `==` and `>`.
-/// - duality: `a < b` if and only if `b > a`.
+/// - **Transitivity**: if `A: PartialOrd<B>` and `B: PartialOrd<C>` and `A:
+///   PartialOrd<C>`, then `a < b` and `b < c` implies `a < c`. The same must hold for both `==` and `>`.
+///   This must also work for longer chains, such as when `A: PartialOrd<B>`, `B: PartialOrd<C>`,
```

Comment:
```suggestion
///   This should also work for longer chains, such as when `A: PartialOrd<B>`, `B: PartialOrd<C>`,
```

(Acknowledging the current state of things here.)

Reply:
The current state is that this is a must, no? "The comparison *must* satisfy, for all [...]".

Reply:
(Consolidating this to the similar discussion on `PartialEq`.)